end_to_end/3-mitigate-bias-train-model2-registry-e2e.ipynb

```diff
@@ -0,0 +1,1340 @@
+{
```

Comment:
<p>CI is reduced/resolved, and can you also show how this improves the model accuracy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Good idea...I will discuss as follows: Training to achieve over 90 percent classification accuracy, may be easily possible on an imbalanced classification problem.</p><p>Thus, expectations developed regarding classification accuracy that are in reality contingent on balanced class distributions will lead to wrong, misleading assumptions and conclusions : misleading the data scientist and  viewers into believing  that a model has extremely  performance when , actually, it does not.</p>